store/rootmultistore_test.go

```diff
@@ -11,45 +11,50 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+const useDebugDB = false
+
 func TestMultistoreCommitLoad(t *testing.T) {
-	db := dbm.NewMemDB()
+	var db dbm.DB = dbm.NewMemDB()
+	if useDebugDB {
+		db = dbm.NewDebugDB("CMS", db)
+	}
 	store := newMultiStoreWithMounts(db)
 	err := store.LoadLatestVersion()
 	assert.Nil(t, err)
 
-	// new store has empty last commit
+	// New store has empty last commit.
```

Comment:
Our comment conventions are driving me nuts - we should stick with something.  This line is not actually a sentence and thus shouldn’t start with a capital and end with a period, as per many comments in the code - this is what is currently in the coding standards we have. 
I'd contest that this comment as well as several following comments are not sentences. LOL sry this is kind of OCD
https://github.com/tendermint/coding/blob/master/go/coding_standard.md#comments

Reply:
"New store has empty last commit" is a sentence!  It's missing an article but it's got a subject verb object.

Reply:
I think we should just prefer full sentences for most comments, and shortened sentences should be OK too, as long as it is a clear statement.  A non-statement would be, "empty last commit".  But those typically are trailing comments // like this.  Those should be lower-case.

Reply:
I'm going to merge this.  If you still object, lmk, happy to fix the sentences.